### PR TITLE
world-of-tanks: add 'uk' and 'be' to cis region

### PR DESCRIPTION
**Pull request by yurikoles**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

I'm not sure why other languages are spelled in capital letters, for me only 'uk' works.

### CODE REVIEW

File: Casks/world-of-tanks.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
     homepage 'https://worldoftanks.asia/'
   end
 
-  language 'RU' do
+  language 'be', 'ru', 'uk' do
```
Comment (vitorgalvao) [inline Casks/world-of-tanks.rb]: I’ll need some justification, here, this is a weird coupling.

And why would they have both `gb` and `uk`?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/world-of-tanks.rb (hunk 1 above)
Comment (yurikoles) [reply Casks/world-of-tanks.rb]: UK don't works for me, it installs us version. `gb` is for United Kingdom, `uk` is for Ukrainian.

Comment (yurikoles) [reply Casks/world-of-tanks.rb]: @vitorgalvao this couple is for CIS region, users from this region are offered to install Russian version.

Comment (yurikoles) [reply Casks/world-of-tanks.rb]: `be` is for Belarusian.

Comment (vitorgalvao) [reply Casks/world-of-tanks.rb]: `be` is Belgium. Belarus is `by` and Ukraine is `ua`.

Comment (yurikoles) [reply Casks/world-of-tanks.rb]: We talk about different things: countries vs. languages.

Comment (vitorgalvao) [reply Casks/world-of-tanks.rb]: `BE` is already at the top. Why should `BE` and `UK` be here?

Also, change the capitalisation to uppercase, it works fine. You also need to capitalise when giving `--language`.

**Comment by reitermarkus** [comment]
Regions should be uppercase.

**Comment by reitermarkus** [comment]
And `uk` should be `GB`.

**Comment by yurikoles** [comment]
bump?

File: Casks/world-of-tanks.rb [hunk 2]
```diff
@@ -26,7 +26,7 @@
     homepage 'https://worldoftanks.asia/'
   end
 
-  language 'RU' do
+  language 'AZ', 'BE', 'HY', 'KK', 'KY', 'RU', 'UK', 'UZ', 'TG' do # CIS region
```
Comment (vitorgalvao) [inline Casks/world-of-tanks.rb]: Please remove the comment.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
> Checksum for Cask 'world-of-tanks' does not match.
>
> Expected:133b3f3271eb957a4eea59cd59a6f79d2c4376a4297a980d8c9232ae9419a995
> Actual:23c2e97b89eeabd45aa5a6250e87b15c9f17757271f615e013453fea3949ac60

File: Casks/world-of-tanks.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
   version '2.0.21'
```
Comment (vitorgalvao) [inline Casks/world-of-tanks.rb]: It’s now at `2.0.28`.

**Comment by yurikoles** [comment]
There is mention of version `2.0.28` in all appcasts, but only CIS binary has updated 😕.

File: Casks/world-of-tanks.rb (hunk 3 above)
Comment (yurikoles) [reply Casks/world-of-tanks.rb]: @vitorgalvao, thanks, I had already figured out. See my comment below.

**Comment by yurikoles** [comment]
Oh, asian binary was also updated 😕.

**Comment by yurikoles** [comment]
I had updated committed message to include all languages that I had added to CIS region.

File: Casks/world-of-tanks.rb (hunk 2 above)
Comment (yurikoles) [reply Casks/world-of-tanks.rb]: @vitorgalvao, done.

**Comment by vitorgalvao** [comment]
Travis passed. Merging. Thank you.

**Pull request merged**